src/plugins/timeConductor/ConductorHistory.vue

```diff
@@ -163,6 +176,23 @@ export default {
                 };
             });
         },
+        getDuration(numericDuration) {
+            let result;
+            let age;
+
+            if (numericDuration > TWENTYFOUR_HOURS) {
+                age = (numericDuration / TWENTYFOUR_HOURS).toFixed(2);
+                result = ` for ${age} days`;
```

Comment:
remove space before ` for....` as its already in `${startTime} ${this.getDuration....`